src/view.cpp

```diff
@@ -359,8 +359,16 @@ void view::open_in_browser(const std::string& url) {
 		cmdline.append("'");
 	}
 	stfl::reset();
-	utils::run_interactively(cmdline, "view::open_in_browser");
+	// store the shell return value
+	int status_browser = utils::run_interactively(cmdline, "view::open_in_browser");
```

Comment:
"Exit code" is a common term for the value that a program returns. Please use it instead of "status" here.